### ISSUE TITLE
is it possible to use an array as input?

### ISSUE THREAD

**Issue by emilianos** [trekawek/radioblock#4]
Hi,
it looks really good, I was wondering, do you think it would be possible to use an array of ads using the same format `commercial-start / commercial-end` ?
I'm willing to manually create these audios for my station.. (record ads, crop a few seconds from the start and ending and save in raw format)

**Comment by trekawek** [comment]
Hello @emilianos,

right now the analyzer expects that the jingles appears one after another. It doesn't compare all the passed samples, but only the current one, in a round-robin fashion.

It should be possible to change this behaviour, by modifying this method:

https://github.com/trekawek/radioblock/blob/fc921f23bc409fb11c8d6bc87eabc81a6f79b438/analyzer/src/main/java/eu/rekawek/analyzer/Analyzer.java#L135-L165

Basically, it should be invoked for every `jingleIndex`. I'm not sure about the performance, though.

**Comment by emilianos** [comment]
That's what I thought, thanks for the reply!
